addons/vagrant/wireless.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Configure Wireless Linux
+  hosts: wireless
+  become: true
+  gather_facts: true
+  
+  collections:
+    - inverse_inc.wireless
+
+  pre_tasks:
+  - name: wait for SSH available before provisioning
+    wait_for_connection:
```

Comment:
Not necessary, I used it only for Cumulus to be sure machine is reachable after reboot.